security-self-review.md

```diff
@@ -0,0 +1,81 @@
+# NativeIO Security Self-Review
+
+https://www.w3.org/TR/2020/NOTE-security-privacy-questionnaire-20200508/
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+Since this feature is a storage API, it does not expose any information that is
+not first created by the origin.
+
+## 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes, we only expose data that was persisted by the origin.
+
+## 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+This specification does not expose any personal information.
+
+## 2.4. How does this specification deal with sensitive information?
+
+This specification does not deal with sensitive information.
+
+## 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Yes, this specification introduces a new persistent storage mechanism where
+state may be kept. The risk of tracking must be mitigated by following similar
+strategies used in other storage APIs e.g. user agents should allow users to
+wipe stored data, an origin may only access files that are owned by it, etc.
```

Comment:
"..., access in third-party contexts should be governed by the same policy as other storage types, etc."

Reply:
Done